<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>随机抽选设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="minNum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="minNumText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入最小值：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxNum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="maxNumText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入最大值：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>应用更改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>